cql3/Cql.g

```diff
@@ -1396,8 +1396,9 @@ columnOperation[operations_type& operations]
 
 columnOperationDifferentiator[operations_type& operations, ::shared_ptr<cql3::column_identifier::raw> key]
     : '=' normalColumnOperation[operations, key]
-    | '[' k=term ']' specializedColumnOperation[operations, key, k, false]
-    | '[' K_SCYLLA_TIMEUUID_LIST_INDEX '(' k=term ')' ']' specializedColumnOperation[operations, key, k, true]
+    | '[' k=term ']' collectionColumnOperation[operations, key, k, false]
+    | '.' field=ident udtColumnOperation[operations, key, field]
```

Comment:
Why this rename?

Reply:
Because "collectionColumnOperation" explains more than "specializedColumnOperation" IMO. And now there are two types of "specialized" operations: for collections and for UDTs, and we have to distinguish between them somehow.

Reply:
Maybe collectionElementOperation is more descriptive.